### PR TITLE
Add license

### PR DESCRIPTION
**Pull request by rebornix**
```
GitHubRepository> Unable to fetch mentionable users: Error: GraphQL error: Something went wrong while executing your query. Please include `4798:2E6F:A16534:C78174:5C523C7A` when reporting this issue.

```

### CODE REVIEW

**Comment by sandy081** [comment]
```diff
diff --git a/README.md b/README.md
index a11e31c..aef8f3e 100644
--- a/README.md
+++ b/README.md
@@ -2,4 +2,6 @@
 
 ## License
 
-MIT
\ No newline at end of file
+MIT
+
+Some change
\ No newline at end of file

```

**Comment by sandy081** [comment]
```diff
diff --git a/README.md b/README.md
index a11e31c..3804d38 100644
--- a/README.md
+++ b/README.md
@@ -2,4 +2,6 @@
 
 ## License
 
+Suggest some changes
+
 MIT
\ No newline at end of file

```

**Comment by sandy081** [comment]
This is some comment

File: .vscode/settings.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
     "appService.zipIgnorePattern": [
         "node_modules{,/**}"
-    ]
+	],
```
Comment (RMacfarlane) [inline .vscode/settings.json]: hello

File: .vscode/settings.json [hunk 2]
```diff
@@ -1,5 +1,6 @@
 {
     "appService.zipIgnorePattern": [
```
Comment (RMacfarlane) [inline .vscode/settings.json]: test

File: .vscode/settings.json [hunk 3]
```diff
@@ -1,5 +1,6 @@
...
         "node_modules{,/**}"
-    ]
+	],
+	"editor.insertSpaces": true
 }
```
Comment (RMacfarlane) [inline .vscode/settings.json]: asdf

File: .vscode/settings.json [hunk 4]
```diff
@@ -1,5 +1,6 @@
 {
     "appService.zipIgnorePattern": [
         "node_modules{,/**}"
-    ]
+	],
+	"editor.insertSpaces": true
```
Comment (queerviolet) [inline .vscode/settings.json]: Hi

File: .vscode/settings.json (hunk 3 above)
Comment (queerviolet) [inline .vscode/settings.json]: LGTM

Comment (queerviolet) [inline .vscode/settings.json]: hjh

File: README.md [hunk 5]
```diff
@@ -1 +1,5 @@
-# nodejs-shopping-cart
\ No newline at end of file
+# nodejs-shopping-cart
+
```
Comment (queerviolet) [inline README.md]: a

File: README.md [hunk 6]
```diff
@@ -1 +1,5 @@
...
+## License
```
Comment (queerviolet) [inline README.md]: n

File: README.md [hunk 7]
```diff
@@ -1 +1,5 @@
...
+
+MIT
```
Comment (queerviolet) [inline README.md]: asdfadfasdf

Comment (queerviolet) [inline README.md]: asdfasdfasdf

**Comment by RMacfarlane** [comment]
New comment

File: .vscode/settings.json (hunk 3 above)
Comment (rebornix) [reply .vscode/settings.json]: great!

File: README.md (hunk 5 above)
Comment (rebornix) [reply README.md]: b

Comment (rebornix) [reply README.md]: c

Comment (rebornix) [reply README.md]: d

File: .vscode/settings.json [hunk 8]
```diff
@@ -1,5 +1,6 @@
 {
     "appService.zipIgnorePattern": [
         "node_modules{,/**}"
```
Comment (RMacfarlane) [inline .vscode/settings.json]: add new pending review

File: .vscode/settings.json (hunk 4 above)
Comment (RMacfarlane) [inline .vscode/settings.json]: add comment to pending

File: .vscode/settings.json (hunk 2 above)
Comment (RMacfarlane) [inline .vscode/settings.json]: newer review

Comment (RMacfarlane) [inline .vscode/settings.json]: review

Comment (RMacfarlane) [inline .vscode/settings.json]: changes

File: .vscode/settings.json (hunk 8 above)
Comment (RMacfarlane) [inline .vscode/settings.json]: review

Comment (RMacfarlane) [inline .vscode/settings.json]: review

File: .vscode/settings.json [hunk 9]
```diff
@@ -1,5 +1,6 @@
...
-    ]
```
Comment (RMacfarlane) [inline .vscode/settings.json]: asdf

Comment (RMacfarlane) [inline .vscode/settings.json]: pending

File: .vscode/settings.json (hunk 8 above)
Comment (RMacfarlane) [inline .vscode/settings.json]: asdf

Comment (RMacfarlane) [inline .vscode/settings.json]: test

Comment (RMacfarlane) [reply .vscode/settings.json]: retest

Comment (RMacfarlane) [inline .vscode/settings.json]: test

Comment (RMacfarlane) [inline .vscode/settings.json]: test

Comment (RMacfarlane) [inline .vscode/settings.json]: kjlhkj

**Comment by rebornix** [comment]
@rebornix

File: .vscode/settings.json (hunk 8 above)
Comment (RMacfarlane) [reply .vscode/settings.json]: asdasdf

**Comment by isidorn** [comment]
Test

File: .vscode/settings.json (hunk 3 above)
Comment (isidorn) [reply .vscode/settings.json]: Adding a comment

Comment (isidorn) [reply .vscode/settings.json]: Asdasadsad

File: .vscode/settings.json (hunk 8 above)
Comment (isidorn) [reply .vscode/settings.json]: From GITHUB UX

Comment (isidorn) [reply .vscode/settings.json]: dasdasdas

Comment (isidorn) [reply .vscode/settings.json]: sdadsa

Comment (isidorn) [reply .vscode/settings.json]: adadsdasdasadssaadsasd

Comment (isidorn) [reply .vscode/settings.json]: dsadsaads

Comment (isidorn) [reply .vscode/settings.json]: sdadsadsasda

Comment (isidorn) [reply .vscode/settings.json]: Latest COMMENT

Comment (isidorn) [reply .vscode/settings.json]: Latest COMMENT

Comment (isidorn) [reply .vscode/settings.json]: Double comment

Comment (isidorn) [reply .vscode/settings.json]: Double comment

Comment (isidorn) [reply .vscode/settings.json]: Double hello

Comment (isidorn) [reply .vscode/settings.json]: Double hello

**Comment by misolori** [comment]
LGTM!

File: .vscode/settings.json [hunk 10]
```diff
@@ -1,5 +1,6 @@
 {
```
Comment (RMacfarlane) [inline .vscode/settings.json]: great

File: README.md (hunk 5 above)
Comment (rebornix) [reply README.md]: def

Comment (rebornix) [reply README.md]: hijk

Comment (rebornix) [reply README.md]: lmn

Comment (rebornix) [reply README.md]: opq

Comment (rebornix) [reply README.md]: start my review.

Comment (rebornix) [reply README.md]: start my second review.

File: .vscode/settings.json (hunk 2 above)
Comment (RMacfarlane) [inline .vscode/settings.json]: hmm

Comment (RMacfarlane) [inline .vscode/settings.json]: asdfasdfasdf

File: README.md (hunk 5 above)
Comment (rebornix) [reply README.md]: my review

Comment (rebornix) [reply README.md]: asdfasdf

Comment (rebornix) [reply README.md]: ;)

Comment (rebornix) [reply README.md]: abc

Comment (rebornix) [reply README.md]: abc

Comment (rebornix) [reply README.md]: abc

Comment (rebornix) [reply README.md]: ~

Comment (rebornix) [reply README.md]:
~~~

Comment (rebornix) [reply README.md]: !!

File: .vscode/settings.json (hunk 2 above)
Comment (RMacfarlane) [inline .vscode/settings.json]: great!

**Pull request closed without merging**